osbuild-composer.spec

```diff
@@ -22,7 +22,7 @@ It is compatible with composer-cli and cockpit-composer clients.
 }
 
 Name:           osbuild-composer
-Release:        1%{?dist}
```

Comment:
The release in the upstream spec is ignored. Please add `.1` to the version instead.